### DISCUSSION TITLE
can not finish Neptuns revenge - campaign

### DISCUSSION THREAD

**Discussion by fik236** [widelands/widelands#5702] [General]
I can not finish campaign Neptuns revenge.  Almost whole map is under my control
see printscreen http://fiksoftware.4fan.cz/zasek.png
Problem is that I need this last place of oponents harbor, because it is near the last part of artefact.

I can not send there any solider to  conquere it. Even if I have there cosest as possible tower full of my soliders.
And I can no also mine other stones (even if I replace my tower with quarry).

Thank you for any advice.

**Comment by klaus-halfmann** [comment]
Good catch, normally the AI does not build a harbor there (But you are supposed to do so :-) 
A scout will not help you, as it cannot cross those stones either. Please attach a savegame
and tell us which version of widelands this is.

I Remember it took me about 3 approaches until I could finish this one, so how often did you 
retry? 

OTOH Did you ever use the Map Editor? You could extract and then modify the underlying Map,
so that a single extra building can be placed there. But then the Barbarians could collect all
the stones there, mmh.

I do not consider this a real bug, yet.

**Comment by fik236** [comment]
Hello, thank you for reply.
I am running widelans version 1.1 as AppImage under Xubuntu 22.04

I Can not find where is savegame stored.

**Comment by klaus-halfmann** [comment]
You could find you savegame at `.widelands/saves` Not sure if this appllies for an Appmage, too

**Comment by fik236** [comment]
This folder does exists in my system. But I have got an idea - run app image trough terminal and thanks output I have found saved games in my home folder in: .local/share/widelands/save/

so my problematic save game is: [fiksoftware.4fan.cz/zasek.wgf](url)

ps.: exist some "normal" way hot to add to this discusion some attachment?

**Reply by hessenfarmer** [reply]
the link does not work for me. please provide the save again

**Comment by klaus-halfmann** [comment]
Hello fik, you can just drag a file into the input field, but you need to zip (or rename) it.
Did you find the MapEditor? If all fails you can download the scenario from e.g. here:
https://github.com/widelands/widelands/tree/master/data/campaigns/emp02.wmf 
I will take a look at your savefile meanwhile.

**Comment by klaus-halfmann** [comment]
You know you can tear down mititary builldíngs? that are not anywhere near the enemy?

**Comment by klaus-halfmann** [comment]
I tried to get rid of the stones. I will try to edit that map, but this will take some time.

**Comment by hessenfarmer** [comment]
this should have been filed as an issue. As it is a bug uin the scenario.   
@Noordfrees is there any tool in github to move this from discussions to issues?

**Reply by klaus-halfmann** [reply]
@hessenfarmer the original text as link works, the _underlying_ link does not. 
I already tried whatever I could imagine but still are stuck in the same way.

**Comment by klaus-halfmann** [comment]
Help is on the way, for the next version, https://github.com/widelands/widelands/pull/5722
